tree/dataframe/test/dataframe_regression_tests.hxx

```diff
@@ -106,3 +109,40 @@ TEST(TEST_CATEGORY, UniqueEntryNumbers)
 
    gSystem->Unlink(fname);
 }
+
+// ROOT-9731
+TEST(TEST_CATEGORY, ReadWriteTVector3)
+{
+   const std::string filename = "readwritetvector3.root";
+   {
+      TFile tfile(filename.c_str(), "recreate");
+      TTree tree("t", "t");
+      auto a = TVector3();
```

Comment:
Why is this not
```suggestion
     TVector3 a;
```
? Obviously doesn't make a difference...

Reply:
style preference on the part of the original author I guess